cmd/view.go

```diff
@@ -48,24 +53,74 @@ func viewCmd(ctx *cli.Context) error {
 		return err
 	}
 
-	verbose := ctx.Bool("verbose")
-	if verbose {
-		fmt.Printf("Credential path: %s\n\n", path)
+	format := ctx.String("format")
+	if ctx.Bool("verbose") {
+		format = "verbose"
+	}
```

Comment:
Our existing commands error when you specify two that do similar things. IE `services list`. That's the best way to go here too, IMO, as `--format json --verbose` reads like it should show verbose json output (which we might want to do some day!)